lib/msal-common/src/request/ScopeSet.ts

```diff
@@ -61,62 +77,84 @@ export class ScopeSet {
         this.appendScope(Constants.OFFLINE_ACCESS_SCOPE);
     }
 
+    /**
+     * Replace client id with the default scopes used for token acquisition.
+     */
+    private addDefaultScopes(): void {
```

Comment:
Will the node library get an id token by default?

Reply:
Yes, needed for the cache.